content/fr/case-studies/0.github.md

```diff
@@ -1,55 +1,55 @@
+GitHub est le foyer de millions de développeurs. Nous avons plus de 65 millions de développeurs et c'est essentiellement un endroit où les développeurs peuvent aller créer, partager et rendre leur meilleur code possible. Nous essayons de permettre aux développeurs de rencontrer facilement des développeurs, de travailler ensemble, de résoudre des problèmes difficiles et de créer les technologies les plus importantes au monde. Notre communauté est composée d'un ensemble diversifié de personnes allant des étudiants aux amateurs, des professionnels de l'entreprise, des partenaires et des cadres, et la liste est longue. GitHub n'est pas seulement du code, c'est bien plus que du code, c'est le foyer de la collaboration open source. C'est là que les nouveaux développeurs peuvent venir et commencer. C'est là que les développeurs expérimentés peuvent approfondir leurs connaissances en résolvant les problèmes insolubles et en testant les limites de ce que les logiciels peuvent faire.
 
-## Is GitHub open sourced?
+## GitHub est-il open source ?
 
-It is an idea that is worth considering and there are parts of GitHub across infrastructure tooling that are already open source. At github.com/github you will be able to see all of the open source projects that we have there. Two notable examples are:
+C'est une idée qui mérite d'être considérée et il y a des parties de GitHub à travers les outils d'infrastructure qui sont déjà open source. Sur [github.com/github](https://github.com/github/), vous pourrez voir tous les projets open source que nous avons là-bas. Deux exemples notables sont :
 
-  The [GitHub load balancer](https://github.blog/2018-08-08-glb-director-open-source-load-balancer/) which we open sourced back in 2018. It is our scalable load balancer solution for the bare metal data centres and it powers the majority of GitHub's web and git traffic as well as it is fronting some of the principal intel systems.
+  Le [load balancer GitHub](https://github.blog/2018-08-08-glb-director-open-source-load-balancer/) que nous avons open sourcé en 2018. C'est notre solution d'équilibreur de charge évolutive pour les centres de données bare metal et il alimente la majorité du trafic web et git de GitHub ainsi qu'il fait face à certains des principaux systèmes Intel. 
 
-  [GitHub Docs](https://github.blog/2020-10-14-how-we-open-sourced-docs-github-com/) which we open sourced in mid October 2020. With this effort, we're able to source new ideas from a broader and diverse set of individuals, especially from those who are  experts in the community and GitHub.
+  [GitHub Docs](https://github.blog/2020-10-14-how-we-open-sourced-docs-github-com/) que nous avons open sourcé en mi-octobre 2020. Avec cet effort, nous sommes capable de trouver de nouvelles idées auprès d'un ensemble plus large et diversifié d'individus, en particulier de ceux qui sont des experts de la communauté et de GitHub.
 
-## What is GitHub Stars?
+## Quest-ce que GitHub Stars?
```

Comment:
missing white space after "Start"

Reply:
@roiLeo Thanks for this :) 
Feel free next time to `Insert suggestion`. More efficient to accept your purpose 😇